helm/chart/maesh/crds/smi-trafficsplits.yaml

```diff
@@ -5,21 +5,67 @@ metadata:
   name: trafficsplits.split.smi-spec.io
 spec:
   group: split.smi-spec.io
-  version: v1alpha2
```

Comment:
To be consistent with the previous CRD file names, we should rename this file to `smi-split.yaml`.
wdyt?